package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "react_phone-catalog",
-  "homepage": "react_phone-catalog",
+  "homepage": "https://mixelio.github.io/react_phone-catalog",
```

That might be the reason of problem, try this:

```suggestion
  "homepage": ".",
```